<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AggregateOptionsWidget</class>
 <widget class="QWidget" name="AggregateOptionsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>553</width>
    <height>53</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="1" column="0">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>Grouping:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Method:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QRadioButton" name="rbHourly">
       <property name="toolTip">
        <string>60 minutes</string>
       </property>
       <property name="text">
        <string>Hourly</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rbDaily">
       <property name="toolTip">
        <string>1440 minutes</string>
       </property>
       <property name="text">
        <string>Daily</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rbWeekly">
       <property name="toolTip">
        <string>10080 minutes</string>
       </property>
       <property name="text">
        <string>Weekly</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rbMonthly">
       <property name="text">
        <string>Monthly</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rbYearly">
       <property name="toolTip">
        <string/>
       </property>
       <property name="text">
        <string>Yearly</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rbCustom">
       <property name="text">
        <string>Custom:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="sbCustomMinutes">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="suffix">
        <string> minutes</string>
       </property>
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>9999999</number>
       </property>
       <property name="value">
        <number>1051898</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="cbMethod"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>rbCustom</sender>
   <signal>toggled(bool)</signal>
   <receiver>sbCustomMinutes</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>85</x>
     <y>61</y>
    </hint>
    <hint type="destinationlabel">
     <x>176</x>
     <y>61</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
